CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-express ChangeLog
 
+## 8.3.0 - 2024-mm-dd
```

Comment:
@dlongley's the one who suggested a point release.

I do think a major release would be warranted typically. In this specific case, we're going from a more restrictive to a more permissive license, so not likely to get complaints/concerns. However, an "opt-in" upgrade step would be a Good Thing.

@dlongley any objections?

Reply:
I don't think a major bump is required -- and a major bump is very disruptive. There are no API changes and we're loosening restrictions, not tightening. If major bumps came at no cost, I think it would be fine, we need to balance whether that's worth it. I doubt it matters at all to pay the price.

Reply:
Another way of looking at this -- is that we could allow anyone to continue using this package under the existing license OR use the new one. If that helps move this along without mixing the idea of API breakage ("major changes") with license changes (that in this case are less restrictive), I'm +1 to that. There's nothing wrong with someone using this package using the old license -- or the new less restrictive one, IMO.

Reply:
Agreed. It's why I didn't push for a major version bump--because this change doesn't materially effect anyone using it.